tests/helpers/run-command.js

```diff
@@ -111,4 +94,15 @@ module.exports = function run(/* command, args, options */) {
       }
     });
   });
+
+  promise.kill = function() {
```

Comment:
A broader refactoring of `runCommand` is needed, but I worry about doing that in this PR as we intend to as a bugfix + backport.

Tracking `runCommand` related improvements here: https://github.com/ember-cli/ember-cli/issues/9073#issue-573121155